app/views/scaptimony_policies/index.html.erb

```diff
@@ -2,7 +2,7 @@
 <% title _("Compliance Policies") %>
 
 <% title_actions(
-  display_link_if_authorized(_("New Compliance Policy"), hash_for_new_scaptimony_policy_path),
+   display_link_if_authorized(_("New Compliance Policy"), hash_for_new_scaptimony_policy_path),
```

Comment:
I have it set so it can touch/auto-format only lines that were modified by me in a given commit